vzstorage-pd/virtuozzo-provisioner.go

```diff
@@ -139,6 +254,15 @@ func (p *vzFSProvisioner) Delete(volume *v1.PersistentVolume) error {
 	}
 
 	options := volume.Spec.PersistentVolumeSource.FlexVolume.Options
+
+	secret := map[string][]byte{
+		"clusterName":     []byte(options["clusterName"]),
```

Comment:
Are you sure that options contain clasterName and clusterPassword? Who added them to options? How does kube describe volume show them?